substrate/frame/timestamp/src/lib.rs

```diff
@@ -135,14 +158,17 @@ pub mod pallet {
 			+ MaxEncodedLen
 			+ scale_info::StaticTypeInfo;
 
-		/// Something which can be notified when the timestamp is set. Set this to `()` if not
-		/// needed.
+		/// A means of notifying another pallet when the timestamp is set.
```

Comment:
I preferred the old comment, because you set "something" that can be notified. You could say it's always a pallet to be notified, but that's not necessarily the case.